scripts/lint-style.py

```diff
@@ -350,6 +351,13 @@ def left_arrow_check(lines, path):
         newlines.append((line_nr, new_line))
     return errors, newlines
 
+def adaptation_note_check(lines, path):
+    errors = []
+    for line_nr, line in lines:
+        if "Adaptation note:" in line:
```

Comment:
Is this check safer? Because of caps and/or missing colons?
```suggestion
        if "daptation note" in line:
```

Reply:
Good idea! Did this and added a comment about this.